public/core.d.ts

```diff
@@ -8808,48 +8808,942 @@ declare namespace JMap {
+     * 
+     * This is a technical method used by JMap App, you should never used this function
+     * 
+     * @example ```ts
+     * 
+     *  const form = ...
+     *  // returns the form values
+     *  JMap.Form.getFormValues(form)
+     * ```
+     */
     function getFormValues(form: JForm, initialData?: JAttributeValueByName): JAttributeValueByName
+
+    /**
+     * ***JMap.Form.setFormValues***
+     * 
+     * Set form values of the given form.
+     * 
+     * @example ```ts
+     * 
+     *  const form = JMap.Form.getAttributeForm()
+     *  // set values of the attribute form
+     *  JMap.Form.setFormValues(form, {
+     *    color: "white",
+     *    type: "tree"
+     *  })
+     * ```
+     */
     function setFormValues(form: JForm, attributeValueByName: JAttributeValueByName): JFormErrors
+
+    /**
+     * ***JMap.Form.submit***
+     * 
+     * Set form values of the given form.
+     * 
+     * @throws if no form displayed
+     * @param params optional params
+     * @example ```ts
+     * 
+     *  const layerId = 3
+     *  const featureId = 4
+     *  const feature = await JMap.Feature.getById(layerId, featureId)
+     *  // opens form update dialog for one element of layer id=3
+     *  await JMap.Form.openUpdateDialogForLayer(
+     *    layerId,
+     *    [{
+     *      id: featureId,
+     *      attributeValueByName: feature.properties
+     *    }]
+     *  )
+     *  const form = JMap.Form.getAttributeForm()
+     *  // change some values of the attribute form
+     *  JMap.Form.setFormValues(form, {
+     *    color: "white",
+     *    type: "tree"
+     *  })
+     *  JMap.Form
+     *    .submit()
+     *    .then(result => console.log("Submit result", result))
+     *    .catch(error => console.error(error))
+     * ```
+     */
     function submit(params?: JFormSubmitParams): Promise<JFormSubmitResult>
+
+    /**
+     * ***JMap.Form.canDeleteCurrentElements***
+     * 
+     * Returns true if the current form attribute elements can be deleted.
+     * 
+     * @returns false if no form is displayed
+     * @example ```ts
+     * 
+     * // returns true if the current form attribute elements can be deleted
+     * JMap.Form.canDeleteCurrentElements()
+     * ```
+     */
     function canDeleteCurrentElements(): boolean
+
+    /**
+     * ***JMap.Form.deleteCurrentElements***
+     * 
+     * Delete the current attribute form elements.
+     * 
+     * Works only for update of attribute form elements.
+     * 
+     * @example ```ts
+     * 
+     * // delete current form attribute elements
+     * JMap.Form
+     *  .deleteCurrentElements()
+     *  then(result => console.info("Delete result", result))
+     * ```
+     */
     function deleteCurrentElements(): Promise<JFormDeleteResult>
+
+    /**
+     * ***JMap.Form.getNextViewId***
+     * 
+     * Returns the next view id.
+     * 
+     * This is a technical method that you should never used.
```

Comment:
that you should never used => that you should never use / that you should never have to use (comme Jean a dit plus bas)